specs/Freeze.md

```diff
@@ -0,0 +1,155 @@
+# Background
+Edge browser has a sleeping tab feature which reduces resource usage when the tab is in the background. We are introducing WebView2 APIs
+to access this feature so that invisible WebView can use less resources. We'd appreciate your feedback.
+
+
+# Description
+You may call the `TryFeezeAsync` API to have the WebView2 consume less memory. This is useful when your Win32 app becomes invisible, or when your Universal Windows Platform app is suspended, during the suspended event handler before completing the suspended event.
+
+# Examples
+## .Net, WinRT
+```c#
+async protected void OnSuspending(object sender, SuspendingEventArgs args)
+{
+    SuspendingDeferral deferral = args.SuspendingOperation.GetDeferral();
+    // Set webView.Visibility to false first.
+    // WebView2 must be invisible for TryFreezeAsync to succeed.
+    webView.Visibility = false;
+    await webView.CoreWebView2.TryFreezeAsync();
```

Comment:
(Separating out comments from previous PR https://github.com/MicrosoftEdge/WebView2Feedback/pull/621/files#r533302500)

Is the name reasonable Freeze/Unfreeze?

Other names considered:
 * **Suspend/Resume**: Our freeze operation isn't the same as suspend/resume as applied to UWP apps. We're halting some execution of script, and discarding some caches and otherwise freeing up some memory without losing end user state. Is it reasonable to reuse Suspend/Resume here?
* **Sleep/Awake**: Sleep is more associated with threads going to sleep which also isn't necessarily happening here.

Reply:
I don't think UWP app model owns Suspend/Resume. There are a few other APIs that use it today, and .Net uses it for the Thread API. Although, this feature does align with UWP app model, so I think that's actually a point of favor of using the same terminology.

Reply:
Sounds like we should name the APIs as TrySuspend/Resume, right?

Reply:
updated to use suspend/resume naming